### PR TITLE
#121 add prettier config

### LINKED ISSUE #121: Add prettier config file
Add prettier for better styling and formatting.

### PR DESCRIPTION
**Pull request by LennartRukower**
Added prettier file and options for better automatic formatting.

See prettier documentation for an in depth explanation of the different options:
https://prettier.io/docs/en/options.html

Closes #121

### CODE REVIEW

**Review by johannesfbraun — APPROVED** [review]
LGTM

**Review by christophtrybek — CHANGES_REQUESTED** [review]
Got a Question regarding the Proxies

File: client/package.json [hunk 1]
```diff
@@ -2,6 +2,8 @@
   "name": "client",
   "version": "0.1.0",
   "private": true,
+  "proxy": "https://localhost:3030",
```
[the remark this answers is not in the document]
Comment (LennartRukower) [reply client/package.json:5]: This is a change from the pull request #115.
It has nothing to do with the prettier config.

**Review by christophtrybek — APPROVED** [review]
LGTM

**Pull request merged**